testing/code/test_excinfo.py

```diff
@@ -925,3 +925,11 @@ def test_repr_traceback_with_unicode(style, encoding):
     repr_traceback = formatter.repr_traceback(e_info)
     assert repr_traceback is not None
 
+@pytest.mark.xfail
+def test_cwd_deleted_(tmpdir):
```

Comment:
Could you use `testdir` instead? This way we can be sure the sub-test is actually failing, while the actual `test_cwd_deleted` then passes.